src/lib/rnp.cpp

```diff
@@ -4049,18 +4049,15 @@ rnp_key_get_revocation(rnp_ffi_t        ffi,
 
 rnp_result_t
 rnp_key_export_revocation(rnp_key_handle_t key,
-                          rnp_output_t     output,
+                          rnp_output_t     external_output,
                           uint32_t         flags,
                           const char *     hash,
                           const char *     code,
                           const char *     reason)
 try {
-    if (!key || !key->ffi || !output) {
+    if (!key || !key->ffi || !external_output) {
         return RNP_ERROR_NULL_POINTER;
     }
-    if (flags) {
```

Comment:
We should check for wrong flags here as well, i.e. that flags doesn't contain anything except `RNP_KEY_EXPORT_ARMORED`. For instance, please see how this is done in `rnp_add_security_rule()`. Btw, `test_ffi_export_revocation()` must check this at line 444, so I surprised why it passes.

Reply:
Addressed.